nomad/nomad_deployer.go

```diff
@@ -169,3 +174,35 @@ func compareJobs(old, new *nomadapi.Job) bool {
 
 	return false
 }
+
+// addTLSInfo will inject Nomad TLS certificate info to updater job if the TLS certificate is provided
+func addTLSInfo(job *nomadapi.Job) {
+	// Always make sure that job.ID matches with the job ID in nomad mustache template on server side
+	// Suggest to use a job name with format "portainer-updater-{uuid}""
```

Comment:
this is specifically for portainer-updater, maybe we should have a boolean on the edge job? `addTLSInfo`?

Reply:
I think it is a good idea. Even though there is the job name `portainer-updater` check, it is still not 100% accurate. Will add a boolean flag for such case

Reply:
As we added the boolean flag for identify the nomad edge job, I think it is unnecessary to implement the proposed idea. I will remove the comments.